lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -473,7 +473,10 @@ class Distribution(object):
                      'HP-UX': ['HPUX'],
                      'Darwin': ['MacOSX'],
                      'FreeBSD': ['FreeBSD', 'TrueOS'],
-                     'ClearLinux': ['Clear Linux OS', 'Clear Linux Mix']}
+                     'ClearLinux': ['Clear Linux OS', 'Clear Linux Mix'],
+                     'DragonFly': ['DragonFly'],
```

Comment:
In order to map specific distributions to a family, there needs to be a list of distributions that belong to the family:
```suggestion
                     'DragonFly': ['DragonflyBSD', 'Gentoo/DragonflyBSD'],
```

Reply:
@samdoran Thank you! Actually I looked into it but I saw the DragonFly is only returning an empty dictionary, so I thought maybe it is it's own OS family map. [DragonFly](https://github.com/ansible/ansible/blob/ffbc9d99de0b9f86116779f70507ca2598a786fb/lib/ansible/module_utils/facts/system/distribution.py#L574)

```
def get_distribution_DragonFly(self):
        return {}
```

Reply:
It looks like the version parsing for DragonFly is unimplemented, so it will use the values returned [here](https://github.com/ansible/ansible/blob/28dcfa985fd3a2fa3bea71d078e73926a2fcd59d/lib/ansible/module_utils/facts/system/distribution.py#L492-L492) which are not really accurate.


My suggestion contained capitalization errors and omitted `DragonFly`. Sorry about that.
```suggestion
                     'DragonFly': ['DragonFly', 'DragonFlyBSD', 'Gentoo/DragonFlyBSD'],
```